src/impi/core.cljs

```diff
@@ -126,6 +126,33 @@
         [w h] (:pixi.render-texture/size texture)]
     (.create js/PIXI.RenderTexture w h mode)))
 
+(defmulti draw-shape! (fn [_ v] (v :pixi.shape/type)))
+
+(defmethod draw-shape! :pixi.shape/circle 
+  [graphics {[x y] :pixi.shape/position
+             radius     :pixi.circle/radius}]
+  (.drawCircle graphics x y radius))
+
+(defmethod draw-shape! :pixi.shape/ellipse 
+  [graphics {[x y]          :pixi.shape/position
+             [width height] :pixi.shape/size}] 
```

Comment:
I had a similar thought, but went with size because the PIXI https://pixijs.github.io/docs/PIXI.Graphics.html#drawEllipse asks for width and height. I would prefer `:pixi.ellipse/radius [10 5]` since it is describe the radius of the axis and not the full length. Also I believe it is still radius not radii because you are describing the length of the radius at the x and y axes.